<launch>
    <arg name="env" default="tsrb"/>
    <arg name="path_file" default="path0.txt"/>
    <arg name="reset" default="true"/>
    <arg name="trials" default="5"/>
    <arg name="robot_init_pose" default="34.0 -6.0 0.0"/>
    <arg name="idle_time" default="3"/>

    <node name="waypoints_navigator" type="waypoints_navigator.py" pkg="closedloop_nav_slam" output="screen" args="--env $(arg env) --path_file $(arg path_file) --reset --trials $(arg trials) --robot_init_pose $(arg robot_init_pose) --idle_time $(arg idle_time)">
        <remap from="/gt_odom" to="/odom_sparse"/>
        <remap from="/et_odom" to="/visual/odom"/>
        <remap from="/gt_pose" to="/lidar/pose"/>
        <remap from="/et_pose" to="/visual/pose"/>
    </node>
</launch>